src/Razor/src/Microsoft.VisualStudio.RazorExtension/UnifiedSettings/razor.registration.json

```diff
@@ -0,0 +1,121 @@
+{
+  "properties": {
+    "textEditor.razor.advanced.formatOnType": {
+      "type": "boolean",
+      "default": true,
+      "title": "@Setting_FormattingOnTypeDisplayName;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "description": "@Setting_FormattingOnTypeDescription;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "migration": {
```

Comment:
This is called "migration", but there are no changes to how we read settings, so I'm guessing this is two-way, and the new settings UI writes to the old settings store?

Is that the long term solution, or do we have to migrate that code eventually?

Reply:
I think long term we'll need to change the way we do this, but the packages needed will be available with 17.9 P1. This works two ways for now and will write to the old location. The downside will be that in unified settings changes will require a restart. 

tldr; this is temporary, we will need to migrate in the future.

Reply:
> in unified settings changes will require a restart

ouch :)

Reply:
Public previews won't have that issue. It's that or start a vs-deps branch for a few weeks :(